diagnostic.md

```diff
@@ -49,7 +55,7 @@ will take the base color and lighten it by the percentage indicated by mixing it
 with white. Write the code to tint `$cornflower` by 20%.
 
 ```scss
-// your response here
+$cornflower: tint($cornflower, 20%);
```

Comment:
Your syntax for tint is correct, but reassigning it to $cornflower is not only also potentially dangerous - by redefining $cornflower, you essentially introduce a breaking point in your CSS; before that line, $cornflower is one color, and after that line, it's another.